cmd/oras/root/cp.go

```diff
@@ -154,38 +154,45 @@ func doCopy(ctx context.Context, src oras.ReadOnlyGraphTarget, dst oras.GraphTar
 		return graph.Referrers(ctx, src, desc, "")
 	}
 
+	const (
+		Exists  = "Exists "
+		Copying = "Copying"
+		Copied  = "Copied "
+		Skipped = "Skipped"
+	)
```

Comment:
nit: it will be more readable if there is a `label` prefix.
```suggestion
	const (
		labelExists  = "Exists "
		labelCopying = "Copying"
		labelCopied  = "Copied "
		labelSkipped = "Skipped"
	)
```

Reply:
Same suggestions for other files

Reply:
How about `promptXXX`? It's used as status prompt in progress output